### DISCUSSION TITLE
File>Send By Email  uses wrong email client

### DISCUSSION THREAD

**Discussion by kegscy** [sumatrapdfreader/sumatrapdf#2586] [Q&A]
Hi,

Am trying to use the Send by email function but it selects the wrong email client. It DOESN'T select the default email client, but instead selects a secondary one that is installed but not running. Is this defined in the registry somewhere ?

Thanks!

**Comment by GitHubRulesOK** [comment]
SumatraPDF should be passing mailto: to the system thus the Windows System should use its current users default, cant test as for over two decades have never set one in any Windows device so its always C:\Windows\System32\mshtml.dll . An app such as ThunderBird should have the means to over-ride and set itself as default.

**Reply by kegscy** [reply]
The App it fires up is eMClient, which is definitely not the default mail app. If I temporarily rename the eMClient executable it fails completely, so its obviously pointing at it from somewhere..... - I need it to use Outlook (the default client)

**Reply by GitHubRulesOK** [reply]
SumatraPDF knows absolutely nothing about where mailto: will go its 100% Windows
except see https://github.com/sumatrapdfreader/sumatrapdf/issues/2244

Related issue https://github.com/sumatrapdfreader/sumatrapdf/discussions/2501 eventually turned out to be the email was correctly routed to default but in the send folder waiting to be sent!

eM Client has to have been installed as default at some time, perhaps as bundleware with the device. 
Check if there is an uninstall option.

**Reply by kegscy** [reply]
Hi,
Yes, eMClient was installed as default at some time in the past. I could uninstall it, but would rather keep it as it has occasional uses. I guess I could uninstall & reinstall....
Thanks

**Reply by GitHubRulesOK** [reply]
See if outlook and eM Client have alternate ways to switch over as default handler?
 you may find you can have both by simply reassigning mailto: protocol between them

**Reply by GitHubRulesOK** [reply]
reasonable instructions lower down on this page, may be worth considering https://www.makeuseof.com/tag/how-to-change-the-default-email-program-for-mailto-links/

**Reply by kegscy** [reply]
Thanks - but the mailto: protocol handler is already set to Outlook !
I think I will need to uninstall
